README.md

```diff
@@ -41,9 +41,9 @@ That's not a question, it's an order! Or more of a friendly offer, really. Kanda
 
 * GitHub [issues tracker](https://github.com/kandanapp/kandan/issues)
 * Twitter [@kandanapp](https://twitter.com/kandanapp)
-* [Kandan Demo](http://kandan-demo.kandanapp.com) (Yes we eat our own DogFood)
+* [Kandan Demo](http://demo.getkandan.com) (Yes we eat our own DogFood)
```

Comment:
kandan-demo.getkandan.com is kind of verbose for a subdomain, so let's just make it demo.getkandan.com instead.